pyproject.toml

```diff
@@ -108,5 +95,5 @@ python =
 [testenv]
 extras =
     dev
-commands = pytest -v --color=yes --cov=bg_atlasapi --cov-report=xml
+commands = pytest -v --color=yes --cov=bg_atlasapi --cov-report=xml -W ignore::DeprecationWarning
```

Comment:
Just for my understanding. Why do we decide to additionally add ignoring a DeprecationWarning here - is something going to be broken or is this an auto-formatting fix?

Reply:
pandas is throwing a depreciation warning that causes our CI tests to fail (see https://github.com/brainglobe/bg-atlasapi/actions/runs/7640045487/job/20814376965#step:2:271). It's a futurewarning, but because our tox environment is set to elevate warnings to errors, CI just fails.

Reply:
Thanks for explaining.
Should we add PyArrow as a dependency instead to avoid breakage when the new major Pandas release happens? (Possibly in a different issue + PR?) The point of having the tox environment set up like that is to fix these things early, I think?